Ibro.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
```

Comment:
This file seems to be dealing with Login, not Registration. Remove it from this PR.